### DISCUSSION TITLE
Remove around_http_request

### DISCUSSION THREAD

**Discussion by krainboltgreene** [vcr/vcr#868] [Ideas]
It causes thread saftey problems and probably isn't that useful.

This is a sounding board for removing the hook and saving us all a lot of stress.

**Comment by dnesteryuk** [comment]
:+1:

**Comment by mrbrdo** [comment]
With https://github.com/vcr/vcr/pull/512 it should be thread safe.
However I am also in favor of this, after working on this PR, since probably 90% of the code in the PR was just to fix `around_http_request`, as well as most time spent.
Or it could at least be extracted into a separate gem.
